include/internal/list.h

```diff
@@ -20,19 +29,31 @@
 # define OSSL_LIST_MEMBER(name, type)                                       \
     struct {                                                                \
         type *next, *prev;                                                  \
+        OSSL_LIST_DBG(struct ossl_list_st_ ## name *list)                   \
     } ossl_list_ ## name
 
 # define DEFINE_LIST_OF(name, type)                                         \
     typedef struct ossl_list_st_ ## name OSSL_LIST(name);                   \
     struct ossl_list_st_ ## name {                                          \
-        type *head, *tail;                                                  \
+        type *alpha, *omega;                                                \
```

Comment:
Why this name change??? The new names are much less helpful than the old names.

Reply:
Existing code uses _head_ and _tail_, without this name change the existing code compiles but bypasses the accessors which isn't ideal.  With the name change, a compilation error occurs instead.

As mentioned previously, the names can and should be reverted **after** the remaining hardwired list code has been updated.